### ISSUE TITLE
XamlListView throw exception

### ISSUE THREAD

**Issue by egmkang** [Microsoft/Windows-universal-samples#632]
System.InvalidCastException: 'Unable to cast object of type 'System.Char' to type 'System.String'.'

   at ListViewSample.SimpleGridViewSample.SimpleGridViewSample_obj10_Bindings.Update_Key(Object obj, Int32 phase)
   at ListViewSample.SimpleGridViewSample.SimpleGridViewSample_obj10_Bindings.Update_(GroupInfoList obj, Int32 phase)
   at ListViewSample.SimpleGridViewSample.SimpleGridViewSample_obj10_Bindings.Update()
   at ListViewSample.SimpleGridViewSample.SimpleGridViewSample_obj10_Bindings.DataContextChangedHandler(FrameworkElement sender, DataContextChangedEventArgs args)

**Comment by jburman** [comment]
Quick fix is to make GetContactsGrouped in Contacts.cs (line 108) select a String key to group on instead of a Char. Also added ToUpper to eliminate a mixed case scenario.

```
var query = from item in GetContacts(numberOfContacts)
                        group item by item.LastName.Substring(0, 1).ToUpper() into g
                        orderby g.Key
                        select new { GroupName = g.Key, Items = g };
```

**Comment by oldnewthing** [comment]
Fixed in [v5.0.4](https://github.com/Microsoft/Windows-universal-samples/tree/v5.0.4)
